### PR TITLE
Req-uests

### PR DESCRIPTION
**Pull request by FoxLisk**
* [x] Add placeholder text in the Current Season contents if there are no brackets, standings or quals yet
* [x] Player page link in Quals table
* [ ] Users page styles
* [x] Combine Quals VoD link and time
* [x] Brackets page: Take race time outside of player link
   * [ ] ...and change styles for the time

im not sure the time needs to have different style. 

image
image
image

### CODE REVIEW

File: http/static/js/season_qualifiers.js [hunk 1]
```diff
@@ -83,45 +91,47 @@ async function do_delete(id) {
...
+    let wrapper = document.getElementById('qualifiers-wrapper');
+    let no_qualifiers = document.getElementById('no-qualifiers');
     let qualifiers = await get_qualifiers(season_id);
     try {
         var obsolete_hidden = true;
         function rebuild() {
             tbody.innerHTML = "";
             let rows = build_rows(qualifiers);
+            if (rows.length == 0) {
+                no_qualifiers.classList.remove('hidden');
```
Comment (RequiemOfSpirit) [inline http/static/js/season_qualifiers.js:109]: Probably should be `add`, here instead

Comment (RequiemOfSpirit) [reply http/static/js/season_qualifiers.js:109]: And I can't believe you're using snake casing in javascript :pensive:

File: http/templates/season_brackets.html.tera [hunk 2]
```diff
@@ -4,11 +4,19 @@
...
+    {% for bracket in brackets %}
+    <li title="Bracket id#{{ bracket.id }}">
+      {{ macros::link(href=bracket.url, text=bracket.name) }}
+    </li>
+    {% endfor %}
+  </ul>
+</div>
+
+{% else %}
+Brackets have not yet been created for this season.
```
Comment (RequiemOfSpirit) [inline http/templates/season_brackets.html.tera:20]: Might look into styling these placeholders as well, but they look good for now

File: http/static/js/season_qualifiers.js (hunk 1 above)
Comment (FoxLisk) [reply http/static/js/season_qualifiers.js:109]: oh yeah i always forget about that

**Review by RequiemOfSpirit — APPROVED** [review]

**Pull request merged**